llvm/lib/Transforms/IPO/OpenMPOpt.cpp

```diff
@@ -1535,7 +1535,7 @@ struct OpenMPOpt {
     // safely remove it.
     // TODO: This should be somewhere more common in the future.
     if (GlobalVariable *GV = M.getNamedGlobal("__llvm_rpc_client")) {
-      if (GV->getNumUses() >= 1)
+      if (GV->hasNUsesOrMore(1))
```

Comment:
```suggestion
      if (!GV->use_empty())
```
Would be more typical?